src/modules/emote_menu/components/Preview.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Icon from 'rsuite/lib/Icon/index.js';
+import IconButton from 'rsuite/lib/IconButton/index.js';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faStar} from '@fortawesome/free-solid-svg-icons/faStar';
+import styles from '../styles/menu.module.css';
+import emoteStorage from '../stores/emote-storage.js';
+
+function PreviewEmote({emote, ...restProps}) {
+  return (
+    <div {...restProps} key={emote.code}>
+      <div className={styles.content}>
+        <div className={styles.emoji}>
+          <img alt={emote.name} src={emote.images['3x'] || emote.images['2x'] || emote.images['1x']} />
+        </div>
+        <div>
+          <div className={styles.emoteCode}>{emote.code}</div>
+          <p>
+            from <b>{emote.provider.displayName}</b>
+          </p>
+        </div>
+      </div>
+      {emoteStorage.isFavorite(emote.id) && (
```

Comment:
avoid `&&` when doing react renders. otherwise you're sending `false` to the renderer. you are meant to send `null` or a react component.